tests/examples/langchain/medical_rag/test_regression_diagnosis_agent.py

```
@@ -0,0 +1,356 @@
+        "span_id",
+        "trace_id"
+    ]
+    
+    # Check that each vital column is not present in the trace data
+    for column in vital_columns:
+        assert column not in trace_data, f"Expected {column} to be excluded from trace data"
+
+
+if __name__ == "__main__":
+    test_trace_total_cost()
+    test_model_cost_as_no_op()
```

Ruff (0.11.9)

Undefined name `test_model_cost_as_no_op`

(F821)